docs/cli/ctl.md

```diff
@@ -0,0 +1,369 @@
+---
+title: "wash ctl"
+draft: false
+sidebar_position: 6
+description: "wash ctl command reference"
+--- 
+
+This subcommand will help you to interact with the wasmCloud control interface. You may perform operations such as retrieving information about the lattice, enable provisioning or deprovisioning of entities in the lattice, creating links or updating items in the lattice. Following are the subcommands available under `wash ctl`
```

Comment:
I would add a note up top here that the whole `wash ctl` subcommand is deprecated rather than on individual commands